Abcrete/BangazonCLI
# Order

## Status
READY

## Description
Create a order class with necessary properties.  Also added PR template and ERD.png

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. git fetch
2. git checkout order
3. confirm order should have- orderId, customerId, paymentTypeId, dateCreated
4. confirm order class has those properties

## Impacted Areas in Application
Customers & Products

## Link to Feature Ticket
#16
